src/Components/Common/Query/index.js

```diff
@@ -3,7 +3,13 @@
 import React from 'react';
 import { QueryClient, QueryClientProvider } from 'react-query';
 
-const queryClient = new QueryClient();
+const queryClient = new QueryClient({
+  defaultOptions: {
+    queries: {
+      staleTime: 10000,
```

Comment:
I'd keep the default even a bit more agressive personally :)
We never know what data are we going to fetch in the future.
```suggestion
      staleTime: 5000,
```